.github/workflows/compile-examples.yml

```diff
@@ -247,6 +247,7 @@ jobs:
             - name: LoRa
             - name: MKRWAN
             - name: WiFiNINA
+            - name: AceTime
```

Comment:
This causes the `AceTime` library to be installed via Library Manager, and nothing more.

If you want to also compile the AceTime library's examples, you should add their path to `env.UNIVERSAL_SKETCH_PATHS` (I'm guessing there is nothing board-specific about those examples):
https://github.com/arduino/ArduinoCore-samd/blob/738af1784329b3f98330df0e5d3ff59f514753de/.github/workflows/compile-examples.yml#L27
```
        - ~/Arduino/libraries/AceTime/examples
```
I'm happy to submit a PR against your PR branch to add this if you prefer.

Reply:
Yes, thanks! I completely misinterpreted the meaning of that entry :smile:

Reply:
PR submitted: https://github.com/facchinm/ArduinoCore-samd/pull/3